environments/kubernetes/manifests/thanos-rules-prometheusrule.yaml

```diff
@@ -177,11 +176,21 @@ spec:
       annotations:
         message: Thanos Receive {{$labels.job}} has not been able to reload hashring
           configurations.
-      expr: avg(thanos_receive_config_last_reload_successful{job=~"thanos-receive.*",
-        namespace="telemeter-stage"}) by (job) != 1
+      expr: avg(thanos_receive_config_last_reload_successful{job=~"thanos-receive-.*"})
+        by (job) != 1
       for: 5m
       labels:
         severity: warning
+  - name: thanos-sidecar.rules
```

Comment:
We don't have `sidecar` in our setup, I think we should remove this alert.

Reply:
Good catch. I made sure that the SidecarDown alerts are gone. Will take care of this.